### ISSUE TITLE
How to Frozen Partial Weights of Model?

### ISSUE THREAD

**Issue by Gaoxj2020** [open-mmlab/mmocr#828]
Sorry to bother you. I want to finetune the partial part of my model, how to freeze the weights of the specific part such as backbone, encoder, etc.

**Comment by gaotongxiao** [comment]
You can zero out the learning rate of the modules you desire to freeze in the optimizer config. Say that you want to freeze `backbone` and `neck`:
```python
optimizer = dict(
...
    paramwise_cfg=dict(
        custom_keys={
            'backbone': dict(lr_mult=0, decay_mult=0),
            'neck': dict(lr_mult=0, decay_mult=0)
        }))
```
More explanations on the arguments can be found at [MMCV's docs](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.DefaultOptimizerConstructor)

**Comment by Gaoxj2020** [comment]
> You can zero out the learning rate of the modules you desire to freeze in the optimizer config. Say that you want to freeze `backbone` and `neck`:
> 
> ```python
> optimizer = dict(
> ...
>     paramwise_cfg=dict(
>         custom_keys={
>             'backbone': dict(lr_mult=0, decay_mult=0),
>             'neck': dict(lr_mult=0, decay_mult=0)
>         }))
> ```
> 
> More explanations on the arguments can be found at [MMCV's docs](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.DefaultOptimizerConstructor)

Thank you！
